src/alfasim_score/converter/alfacase/_tests/test_output_results.py

```diff
@@ -6,24 +7,41 @@
 from alfasim_score.converter.alfacase.alfasim_score_converter import AlfasimScoreConverter
 
 
+@pytest.mark.parametrize(
+    "case_filename, input_filename, element_name",
+    [
+        pytest.param(
+            "case.data",
+            "score_input_natural_flow.json",
+            "7-SRR-2-RJS (2022-07-28_15-01-27)",
+            id="natural_flow",
+        ),
+        pytest.param(
+            "case2.data", "score_input_B_relief_C_open.json", "WELLBORE", id="B_relief_C_open"
+        ),
+    ],
```

Comment:
You don't need to use `pytest.param`, a tuple is OK, and then add the ids after
```suggestion
    [
        ("case.data", "score_input_natural_flow.json", "7-SRR-2-RJS (2022-07-28_15-01-27)"),
        ("case2.data", "score_input_B_relief_C_open.json", "WELLBORE"),
    ],
    ids=["natural_flow", "B_relief_C_open"]
```

Reply:
Done!